src/components/DashboardLayout/SettingsLeftNav.tsx

```diff
@@ -1,51 +1,110 @@
-import { useContext } from 'react';
+import { useContext, useState, useEffect } from 'react';
 import Button from 'components/Atoms/Button';
-import { faXmark } from '@fortawesome/free-solid-svg-icons';
+import { faXmark, faServer, faRightToBracket } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { BrowserLocationContext } from 'data/BrowserLocationContext';
 import { RadioGroup } from '@headlessui/react';
 import { SettingsContext } from 'data/SettingsContext';
 import Avatar from 'components/Atoms/Avatar';
 import clsx from 'clsx';
 import { useUid } from 'data/UserInfo';
+import CoreSettings from 'components/Settings/CoreSettings';
+import UserSettings from 'components/Settings/UserSettings';
+
+
+export const tabList = [
+    {
+    title: 'General',
+      path: 'general',
+      content: <CoreSettings />,
+    },
+    {
+      title: 'Users',
+      path: 'users',
+      content: <UserSettings />,
+    },
```

Comment:
There's a new "Profile" page Nicholas made last week, it'll be best if we include that here.

in figma, "Account" is a separate category, so there might be some challenges there.
![image](https://user-images.githubusercontent.com/44907675/227733109-df5a6e4d-2607-486f-af7d-8f68cb7f0683.png)